### ISSUE TITLE
Detaching from container doesn't complete until CTRL+C

### ISSUE THREAD

**Issue by pcgeek86** [docker/docker#25845]
# Scenario

- Windows 10 RTM w/ Anniversary Update
- Docker version information below
- Using PowerShell Console Host `powershell.exe`

```
PS C:\Users\TrevorSullivan> docker version
Client:
 Version:      1.12.1-rc1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   7889dc7
 Built:        Fri Aug 12 18:35:53 2016
 OS/Arch:      windows/amd64
 Experimental: true

Server:
 Version:      1.12.1-rc1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   7889dc7
 Built:        Fri Aug 12 18:35:53 2016
 OS/Arch:      linux/amd64
 Experimental: true
```

# Issue Report

When I run an interactive container and try to detach from it, using `CTRL + P, CTRL + Q`, the `docker` CLI doesn't seem to be returning properly, until I manually hit `CTRL + C`.

# Repro Steps

- Create a Windows 10 RTM w/ Anniversary Update instance
- Install **Docker for Windows**
- Open `cmd` or `powershell` console host
- Run an interactive container (eg. `docker run --rm -i -t trevorsullivan/powershell`)
- Use `CTRL + P, CTRL + Q` to detach from the running container
- **NOTICE**: The command prompt does not appear until you hit `CTRL + C`

# Expected Behavior

Detaching from a Docker container using the default `CTRL + P, CTRL + Q` key sequence should properly return to a PowerShell prompt or Windows Command Prompt.

Cheers,
**Trevor Sullivan**
Docker Captain
Microsoft MVP: Cloud & Data Center Management
https://trevorsullivan.net
https://twitter.com/pcgeek86

**Comment by johnstep** [comment]
This is not specific to Windows, and has been fixed per #22938. Thanks!

**Comment by thaJeztah** [comment]
Oh, looks like it indeed, thanks @johnstep, I'll close this issue, but feel free to comment if we closed by mistake
